intellij/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/staticanalysis/IntelliJMarkupModelListener.java

```diff
@@ -85,6 +86,12 @@ public static IntelliJMarkupModelListener initializeAfterAnalysisFinished(
         // Therefore, wait for smart mode and only then start listening, to make sure the codeAnalyzer actually did its thing.
         // For more information see https://www.jetbrains.org/intellij/sdk/docs/basics/indexing_and_psi_stubs.html
         DumbServiceImpl.getInstance(project).runWhenSmart(() -> {
+            // In the case that the analyzer finished after a file was closed just after opening the IDE,
+            // do not try to attach any other listeners. These listeners would fail, as the editor no longer exists.
+            if (Disposer.isDisposed(disposable)) {
+                return;
```

Comment:
Document in the method doc that this method can return null? (And annotate with @Nullabale?)

Reply:
Resolved IRL